iOS Example Frame/iOS Example Frame/Factory/Factory.swift

```diff
@@ -106,10 +106,15 @@ struct Factory {
                                                  supportedSchemes: supportedSchemes,
                                                  billingFormData: billingFormData)
 
-    let style = ThemeDemo.buildCustom2Example()
+      let billingFormStyle = Style.billingForm
 
-    let viewController = PaymentFormFactory.buildViewController(configuration: configuration,
-                                                                style: style,
+      let paymentFormStyle = Style.paymentForm
+
+      let style =  PaymentStyle(paymentFormStyle: paymentFormStyle,
+                                billingFormStyle: billingFormStyle)
+
+      let viewController = PaymentFormFactory.buildViewController(configuration: configuration,
+                                                                  style: style,
```

Comment:
Please undo.

Reply:
This was theme 2 and if you look at line 109 it was using the ThemeDemo object to showcase the code required to use Theme approach :).